base/docs/Docs.jl

```diff
@@ -12,7 +12,7 @@ const modules = Module[]
 
 const META′ = :__META__
 
-meta(mod) = mod.(META′)
+@eval meta(mod) = mod.$META′
```

Comment:
I did a search for related llvm issues but didn't find anything (not so surprising given we're probably one of the heaviest llvm jit user).

@tkelman Were you able to learn anything from the buffer pointers in llvm? (The retry condition is a pointer comparison IIRC and I'm wondering if we can get a hint from how they are changed in the infinite loop.)